src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -132,8 +133,9 @@ public void run() {
         }
 
         private void addConcert(int start, int anz) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 105 to the 15 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3776)